src/Component.svelte

```diff
@@ -7,19 +7,15 @@
 	export let fontSize = '1rem'
 	export let align = 'left'
 	export let checkedColor = '#bbe572'
-	export let borderTop = '1px solid #ddd'
+	export let borderTop = 'none'
 	export let borderBottom = 'none'
-	export let paddingTop = '1em'
-	export let paddingBottom = 'none'
-	export let marginTop = '1.25em'
-	export let marginBottom = 'none'
-
+	export let padding = '1em 0 0 0'
```

Comment:
I think these should be `0` by default. I don't think we should enforce a random top margin or padding.

Reply:
Fair point, I've updated the PR